setup.py

```diff
@@ -61,7 +61,7 @@ def check_setuptools_features():
     # TODO Consider not installing the db drivers, or putting them in extras.
     'rethinkdb~=2.3',  # i.e. a version between 2.3 and 3.0
     'pymongo~=3.4',
-    'pysha3>=0.3',
+    'pysha3==1.0b1',
```

Comment:
Good catch, i think i was confused by the changelog on https://pypi.python.org/pypi/pysha3, looks like they have an error there.